corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
Also, as an aside, I've been trying to come up with good terms for the various types of applications, and "working copy" is the most descriptive name for that I've heard yet (I've tried "canonical version" and "latest save", but not really happy with either).  I'll definitely steal that - props!